sourcegraph/sourcegraph
# [SG-39363] Extract bootstrap style import` @import 'bootstrap/scss/card'` in `client/vscode`

## Description
Removes bootsrap `card` stylesheet import and converted to wildcard component

## Ref
[Gitstart Issue
](https://app.gitstart.com/clients/sourcegraph/tickets/SG-39363)[Sourcegraph Issue](https://github.com/sourcegraph/sourcegraph/issues/39363)

## Success criteria
1. All use cases of `card-*` are migrated to use wildcard's `Card` component.


## Implementation details
1.  Remove bootstrap import @import 'card/scss/card' in `forkedBranded.scss`.
2. Covert use cases of `card-*` className to `Card` wildcard component.
3. If step 2. above is not doable, let's copy style rule from bootstrap to be used internally.

## Test Plan
- Run the vscode app on this branch
- See all use cases of bootstrap's `card-*` className on the codebase (seam to be only in `HomeFooter.tsx`)
- Confirm there are no UI diffs.

## App preview:

- [Web](https://sg-web-contractors-sg-39363.onrender.com)
- [Storybook](https://5f0f381c0e50750022dc6bf7-ejelitfaoe.chromatic.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
